app/static/src/scss/includes/_record.scss

```diff
@@ -257,6 +257,95 @@
   margin-top: -2rem;
 }
 
+.govuk-tabs__panel__record-details {
+  display: flow-root;
+  padding: 1rem !important;
+}
+
+.govuk-tabs__panel {
+  padding: "3rem 0:0";
+}
+
+.js-enabled .govuk-tabs__panel {
+  padding: 0;
+}
+
+.btn {
+  color: white;
+}
+
+.headerPanel .options {
+  border-bottom: 2px solid white;
+}
+
+.uv .mainPanel .leftPanel {
+  border-right: 2px solid white;
+  // border-top: 2px solid white;
+  background-color: #0b0c0c !important;
+  // width: 15rem !important;
+}
+
+.uv .mainPanel {
```

Comment:
any way we can have some of these classes under the same .uv class namespace? as in .uv { .mainPanel ... } just for the sake of it being slightly more organized

Reply:
Updated to move classes under .uv class